components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -768,4 +770,89 @@ - (void)testFullScreenContentLayoutCalculationsComplete {
   // It has no visible side effects, so as long as this test finishes, it passes
 }
 
+- (void)testSettingElevationInDrawerViewControllerPropogatesToPresentationController {
+  // Given
+  self.drawerViewController.elevation = 20;
+
+  // Then
+  MDCBottomDrawerPresentationController *presentationController =
+      (MDCBottomDrawerPresentationController *)self.drawerViewController.presentationController;
+  XCTAssertEqualWithAccuracy(presentationController.elevation, 20, 0.001);
+}
+
+- (void)testCorrectDefaultElevationValue {
+  // When
+  [self.fakeBottomDrawer loadViewIfNeeded];
+
+  // Then
+  XCTAssertEqualWithAccuracy(((MDCShadowLayer *)self.fakeBottomDrawer.shadowedView.layer).elevation,
+                             16, 0.001);
+}
+
+- (void)testElevationIsSetCorrectly {
+  // Given
+  self.fakeBottomDrawer.elevation = 12;
+
+  // When
+  [self.fakeBottomDrawer loadViewIfNeeded];
+
+  // Then
+  XCTAssertEqualWithAccuracy(((MDCShadowLayer *)self.fakeBottomDrawer.shadowedView.layer).elevation,
+                             12, 0.001);
+}
+
+- (void)testSettingElevationInPresentationControllerAfterPresentationWillPropagateCorrectly {
+  // When
+  [self.presentationController presentationTransitionWillBegin];
```

Comment:
Please add one test that is just like this but the elevation is set *before* calling `presentationTransitionWillBegin`.

Reply:
done.